src/util/apiUtils.test.js

```diff
@@ -20,9 +22,9 @@ import {
 	apiResponseDuotoneSetter,
 	buildRequestArgs,
 	errorResponse$,
+	getAuthHeaders,
```

Comment:
My preference is to leave them as explicit imports because it makes it easier to see what `export`ed functions actually have tests, although the coverage report should also be doing that.

It's also partially just a [habit from Python](https://docs.python.org/2/howto/doanddont.html#at-module-level)